src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLSPConstants.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer;
+
+internal static class RazorLSPConstants
+{
+    public const string RazorLanguageServerName = "Razor Language Server Client";
```

Comment:
Can we put this in https://github.com/dotnet/razor/blob/activityid/src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/LanguageServerConstants.cs rather than create another new file for constants? Or is that not accessible at this layer?